public/locales/mn/translation.json

```diff
@@ -0,0 +1,45 @@
+{
+  "website": {
+    "name": "Азиудын эсрэг үзэн ядалтаас үүдсэн гэмт хэргийн бүртгэл"
+  },
+  "location" : "Байршил",
+  "date_range" : "Огнооны муж",
+  "daily_count" : "Өдөр бүрийн тоо",
+  "monthly_count" : "Сар бүрийн тоо",
+  "load_more" : "Үргэлжлүүлэх",
+  "incident_count_by_state" : "Хэргийн тоо аймгаар",
+  "incident_table" : {
+    "total_incident" : "Нийт {{count}} тохиолдол",
+    "no_data" : "Мэдээлэл байхгүй",
+    "total_incident_plural" : "Нийт {{count}} тохиолдлууд",
+    "state" : "Аймаг",
+    "count" : "Тоо",
+    "count_10k_asian" : "Тоо/10K Азиуд",
```

Comment:
10K means 10 thousands. Is there a corresponding translation we can use?

Reply:
Same, replied above.